### PR TITLE
Shell script for test curl commands

### PR DESCRIPTION
**Pull request by pundlikintel**

### CODE REVIEW

File: deployment/sample-workload/execute_workload_flow.sh [hunk 1]
```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+
+function checkIfRequireToolInstalled() {
+    cmdd=$1
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: cmdd is intentional or its typo?

File: deployment/sample-workload/execute_workload_flow.sh [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    content=$(sed '$ d' <<< "$response")  # get all but the last line which contains the status code
+    export $3="$content"
+    eval "$4=$http_code"  
+}
+
+
+#Prerequisite check for curl, jq
+checkIfRequireToolInstalled "curl"
+res=$?
+if [ $res == 1 ]; then
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: can we not use $? here itself?

File: deployment/sample-workload/execute_workload_flow.sh [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+else 
+    echo -e "\u2714   jq is installed"    
+fi
+
+#labels
+label_url="Url"
+label_method="Method"
+label_response="Response"
+label_request="Request"
+label_expected_code="Excpected Code"
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: typo in Expected.

File: deployment/sample-workload/execute_workload_flow.sh [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+#Variable for returning response
+return_var=''
+#Variable for returning code
+return_code=''
+#Get token
+url=$server_addr/taa/v1/token
+
+printf "\n\n"
+printf "\033[1;34m Get attestation token \033[00m\n\n"
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: Attesting workload with Intel Trust Authority.

File: deployment/sample-workload/execute_workload_flow.sh [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+
+#Make key call
+attestationToken=$(jq -r -n --argjson data "$return_var" '$data.attestation_token')
+requestForKey=$( jq -n \
+                  --arg at "$attestationToken" \
+                  --arg kturl "$kbs_host/kbs/v1/keys/$key_id/transfer" \
+                  '{attestation_token: $at, key_transfer_url: $kturl}' )
+url=$server_addr/taa/v1/key
+printf "\n\033[1;34m Get decryption key \033[00m\n\n"
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: Requesting encryption key from KBS.

File: deployment/sample-workload/execute_workload_flow.sh [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+    printf "%*s \033[1;31m%s\033[00m\n" ${#label_expected_code} "$label_response" "| $return_var"
+    exit 1
+fi
+printf "%*s %s\n" ${#label_expected_code} "$label_received_code" "| $return_code"
+printf "%*s %s\n" ${#label_expected_code} "$label_response" "| $return_var"
+printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+
+#make decrypt call
+url=$server_addr/taa/v1/decrypt
+printf "\n\033[1;34m Decrypt model \033[00m\n\n"
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: Decrypting Model using encryption key from KBS.

File: deployment/sample-workload/execute_workload_flow.sh [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+    "blood-glucose": "128",
+    "blood-pressure": "88",
+    "skin-thickness": "39",
+    "insulin": "110",
+    "bmi": "36.5",
+    "dbf": "1.057",
+    "age": "37"
+}'
+url=$server_addr/taa/v1/execute
+printf "\n\033[1;34m Execute model \033[00m\n\n"
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: Executing model with test data.

File: deployment/sample-workload/execute_workload_flow.sh [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+    printf "%*s \033[1;31m%s\033[00m\n" ${#label_expected_code} "$label_response" "| $return_var"
+    exit 1
+fi
+printf "%*s %s\n" ${#label_expected_code} "$label_received_code" "| $return_code"
+printf "%*s %s\n" ${#label_expected_code} "$label_response" "| $return_var"
+printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+
+#reset model
+url=$server_addr/taa/v1/reset
+printf "\n\033[1;34m Reset Model \033[00m\n\n"
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: Clearing decrypted model from memory.

**Review by arvind5 — CHANGES_REQUESTED** [review]

File: deployment/sample-workload/execute_workload_flow.sh (hunk 1 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: Removed.

File: deployment/sample-workload/execute_workload_flow.sh (hunk 2 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: using $?

File: deployment/sample-workload/execute_workload_flow.sh (hunk 3 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: Fixed

File: deployment/sample-workload/execute_workload_flow.sh (hunk 4 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: Done

File: deployment/sample-workload/execute_workload_flow.sh (hunk 5 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: Done

File: deployment/sample-workload/execute_workload_flow.sh (hunk 6 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: Done

File: deployment/sample-workload/execute_workload_flow.sh (hunk 7 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: Done

File: deployment/sample-workload/execute_workload_flow.sh (hunk 8 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: Done

**Review by arvind5 — APPROVED** [review]

**Pull request merged**